zap-known-issues.xml

```diff
@@ -1,30 +1,39 @@
   <riskcode>1</riskcode>
   <confidence>2</confidence>
   <riskdesc>Low (Medium)</riskdesc>
-  <desc>&lt;p&gt;The Anti-MIME-Sniffing header X-Content-Type-Options was not set to &apos;nosniff&apos;. This allows older versions of Internet Explorer and Chrome to perform MIME-sniffing on the response body, potentially causing the response body to be interpreted and displayed as a content type other than the declared content type. Current (early 2014) and legacy versions of Firefox will use the declared content type (if one is set), rather than performing MIME-sniffing.&lt;/p&gt;</desc>
+  <desc>&lt;p&gt;The web/application server is leaking information via one or more &quot;X-Powered-By&quot; HTTP response headers. Access to such information may facilitate attackers identifying other frameworks/components your web application is reliant upon and the vulnerabilities such components may be subject to.&lt;/p&gt;</desc>
   <instances>
   <instance>
   <uri>https://ia-case-documents-api-aat.service.core-compute-aat.internal/v2/api-docs</uri>
   <method>GET</method>
-  <param>X-Content-Type-Options</param>
+  <evidence>X-Powered-By: ASP.NET</evidence>
   </instance>
   <instance>
   <uri>https://ia-case-documents-api-aat.service.core-compute-aat.internal/</uri>
   <method>GET</method>
-  <param>X-Content-Type-Options</param>
+  <evidence>X-Powered-By: ASP.NET</evidence>
+  </instance>
+  <instance>
+  <uri>https://ia-case-documents-api-aat.service.core-compute-aat.internal/asylum/ccdAboutToStart</uri>
+  <method>POST</method>
+  <evidence>X-Powered-By: ASP.NET</evidence>
+  </instance>
+  <instance>
+  <uri>https://ia-case-documents-api-aat.service.core-compute-aat.internal/asylum/ccdAboutToSubmit</uri>
+  <method>POST</method>
+  <evidence>X-Powered-By: ASP.NET</evidence>
   </instance>
   </instances>
-  <count>2</count>
-  <solution>&lt;p&gt;Ensure that the application/web server sets the Content-Type header appropriately, and that it sets the X-Content-Type-Options header to &apos;nosniff&apos; for all web pages.&lt;/p&gt;&lt;p&gt;If possible, ensure that the end user uses a standards-compliant and modern web browser that does not perform MIME-sniffing at all, or that can be directed by the web application/web server to not perform MIME-sniffing.&lt;/p&gt;</solution>
-  <otherinfo>&lt;p&gt;This issue still applies to error type pages (401, 403, 500, etc) as those pages are often still affected by injection issues, in which case there is still concern for browsers sniffing pages away from their actual content type.&lt;/p&gt;&lt;p&gt;At &quot;High&quot; threshold this scanner will not alert on client or server error responses.&lt;/p&gt;</otherinfo>
-  <reference>&lt;p&gt;http://msdn.microsoft.com/en-us/library/ie/gg622941%28v=vs.85%29.aspx&lt;/p&gt;&lt;p&gt;https://www.owasp.org/index.php/List_of_useful_HTTP_headers&lt;/p&gt;</reference>
-  <cweid>16</cweid>
-  <wascid>15</wascid>
```

Comment:
I think we need to just give brief explanation or write comment why we are suppressing.

Reply:
ref link attached